src/styles/blocks/button.scss

```
@@ -0,0 +1,31 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #333333;
+  text-decoration: none;
+  font-family: Sharp Sans No2;
+  height: 60px;
+  width: 100%;
+  font-size: 16px;
+  line-height: 130%;
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+  color: #F2F2F2;
+
+  &:hover {
+    background-color: #E4DFD3;
+    color: #333333;
+    border: 1px solid #333333;
+  }
+
+  @include Tablet() {
+    width: 280px;
+    margin: 0 auto 100px;
+  }
+
+  @include onDesktop {
+    margin: 0;
+  }
+}
```

add newline everywhere :)
```suggestion
}

```